### PR TITLE
Fsi updates 121520b

### PR DESCRIPTION
**Pull request by jbranson000**
#1017 
Contains Card Actions, Card Application Process, Claim Process, Contact Request Details, & Credit Limit Increase Details mixins.

### CODE REVIEW

File: components/mixins/experience-event/experienceevent-card-application-process.schema.json [hunk 1]
```diff
@@ -20,20 +22,15 @@
         "xdm:personalFinances": {
           "type": "object",
           "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-application-process.schema.json:24]: Do we want to keep the nesting "xdm:cardApplicationProcess" ?

File: components/mixins/experience-event/experienceevent-claim-process.schema.json [hunk 2]
```diff
@@ -24,42 +22,37 @@
         "xdm:disputes": {
           "type": "object",
           "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-claim-process.schema.json:24]: Do we need to keep the nesting "disputes" and "claimProcess". We talked about removing nesting with the same property name both as parent and child.

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 3]
```diff
@@ -32,70 +34,65 @@
         "xdm:customerFeedback": {
           "type": "object",
           "properties": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:36]: No nesting here at all ?

File: components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json [hunk 4]
```diff
@@ -20,15 +22,10 @@
         "xdm:personalFinances": {
           "type": "object",
           "properties": {
-            "xdm:creditLimitIncreaseDetails": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json:23]: We still need nesting here right ? "personalFinances" ==> "creditLimitIncreaseDetails" ?

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 5]
```diff
@@ -32,70 +34,65 @@
...
+              "title": "Click to Email",
+              "type": "integer",
+              "description": "Visitor initiates an email to the client from an email link on the site/app."
+            },
+            "xdm:clickToText": {
+              "title": "Click to Text",
+              "type": "integer",
+              "description": "Visitor initiates a SMS/text message to the client from a SMS/text link on the site/app."
+            },
+            "xdm:Subject": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:82]: "Subject" ==> "subject"

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json [hunk 6]
```diff
@@ -32,70 +34,65 @@
...
+              "title": "Chat End",
+              "type": "integer",
+              "description": "Captured when a chat is ended with a visitor."
+            },
+            "xdm:chatOffered": {
+              "title": "Chat Offered",
+              "type": "integer",
+              "description": "Captures the number of times a chat session has been offered to a visitor."
+            },
+            "xdm:chatSessionId": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-contact-request-details.schema.json:52]: ID

File: components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json (hunk 4 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-credit-limit-increase-details.schema.json:23]: Yes, this level of nesting would keep it consistent.

File: components/mixins/experience-event/experienceevent-claim-process.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-claim-process.schema.json:24]: This nesting should not have been included with this branch - I've updated it accordingly.

File: components/mixins/experience-event/experienceevent-contact-request-details.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/mixins/experience-event/experienceevent-contact-request-details.schema.json:36]: Nesting here remains consistent with the other mixins.

File: components/mixins/experience-event/experienceevent-card-application-process.schema.json [hunk 7]
```diff
@@ -20,20 +22,15 @@
         "xdm:personalFinances": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-application-process.schema.json:22]: There is still no nesting here. This "personalFinances" is just a definition not an object.

File: components/mixins/experience-event/experienceevent-card-application-process.schema.json [hunk 8]
```diff
@@ -20,20 +22,15 @@
...
           "type": "object",
```
Comment (mjasrotia) [inline components/mixins/experience-event/experienceevent-card-application-process.schema.json:23]: Also we don't need "type" for a definition

**Review by mjasrotia — APPROVED** [review]

**Review by cmathis — APPROVED** [review]

**Pull request closed without merging**